src/main/java/org/killbill/billing/plugin/adyen/api/mapping/klarna/Account.java

```diff
@@ -0,0 +1,47 @@
+package org.killbill.billing.plugin.adyen.api.mapping.klarna;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Account {
+    @JsonProperty("unique_account_identifier")
+    private String identifier;
+
+    @JsonProperty("account_registration_date")
+    private String registrationDate;
+
+    @JsonProperty("account_last_modified")
+    private String lastModifiedDate;
+
+    public String getIdentifier() {
```

Comment:
Just a suggestion: We could use lombok to generate these getter and setters

Reply:
Ashish, I used IDE to generate the getter/setter methods. Didn't have to write most of it. In this case, the object needs to be serialised to jason with different field name.